truffle-config.js

```diff
@@ -38,19 +20,14 @@ module.exports = {
   },
   compilers: {
     solc: {
-      version: "0.8.4",
+      version: statics.compilers[process.env.WITNET_EVM_REALM || "default"].version || "0.8.6",
```

Comment:
Last clause of this expression may be redundant, but I'm sort of ok with it in this case because `statics.compilers` is defined somewhere else and some future commit may remove the `default` entry from there inadvertently.

On the other hand, this is a bit tricky, because when in the future we change the version here, nothing will ensure that it is also changed in `statics.compilers` :thinking: 

```suggestion
      version: statics.compilers[process.env.WITNET_EVM_REALM || "default"].version,
```

Reply:
Better to remove last clause. If no compiler version is defined at least in `statics.compilers.default.version`, compilation will just fail.